src/styles/watch.scss

```
@@ -0,0 +1,70 @@
+.watch {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  width: #{$watch-size};
+  height: #{$watch-size};
+  border-radius: 50%;
+  background-color: rgb(195, 244, 244);
+  border: 4px solid #7ebfee;
+}
+
+.watch__hand {
+  position: absolute;
+  left: 50%;
+  transform-origin: bottom center;
+  border-radius: (50% 50% 0 0);
+
+  &--minute-hand {
+    top: #{($watch-size - 2 * $minute-hand-height) / 2};
```

DRY - don't repeat yourself
just calc this one time & reuse this variable
Also, you can just use 50%;
```suggestion
    top: $watchCenter;
```

---

second parameter has different meaning, it is not possible to reuse it